packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -514,11 +519,11 @@ export class PerformanceEvent {
 
     public reportEvent(eventNameSuffix: string, props?: ITelemetryProperties, error?: any): void {
         if (!this.event) {
-            this.logger.sendErrorEvent({
-                eventName: "PerformanceEventAfterStop",
-                perfEventName: this.event!.eventName,
+            const errorEvent = {
+                eventName: `${this.eventName}_doubleReport`,
```

Comment:
This new eventName will make this show up in queries with a clause like this:
```
| where Data_eventName has "RefreshLatestSummaryGetSnapshot"
```

Otherwise I fear we'll never proactively look for this event and won't notice when it's happening.

Reply:
I'd keep the old structure, i.e. single event and property to point to type of issue.
This will increase chances that someone will take a look - I usually look at errors via sorting by hit count - if error does not happen that often, it may not even get into my view.

Reply:
Reverted